src/index.js

```diff
@@ -1,14 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import App from "./components/App/";
+import App from "COMPONENTS/App/";
+import { ClientProvider } from "COMPONENTS/Client/";
```

Comment:
Lets' not go back and refactor this, but I think that, for other projects, `import { Provider as ClientProvider } from "COMPONENTS/Client/"` is the way to go here. That way every context exports the same thing - `{ Context, Provider }`. It feels more reliable and sturdy.